federation/pkg/kubefed/init/init.go

```diff
@@ -501,7 +499,7 @@ func createControllerManager(clientset *client.Clientset, namespace, name, cmNam
 							Command: []string{
 								"/hyperkube",
 								"federation-controller-manager",
-								"--master=https://federation-apiserver",
+								fmt.Sprintf("--master=https://%s-apiserver", name),
```

Comment:
The fix for this is already here - https://github.com/kubernetes/kubernetes/pull/36294